src/cbc_sdk/endpoint_standard/models/enrichedEventFacet.yaml

```diff
@@ -0,0 +1,68 @@
+type: object
```

Comment:
This file should be snake_case. The other files in this models folder are old and don't use the new style

Reply:
Done.